doc/api/esm.md

```diff
@@ -394,8 +395,8 @@ from exports subpaths.
 {
   "exports": {
     ".": {
-      "require": "./main.cjs",
-      "default": "./main.js"
+      "import": "./main.js",
```

Comment:
Should we preserve the semantics here and make it:

```js
  ".": [{
    "require": "./main.cjs"
  }, "./main.js"]
```

Or do we think that "make the examples compatible with node 13.1" isn't a goal here?

Reply:
Maybe it's enough to have a quick note saying "hey, if you care about compat with node 13.1, here's a snippet"?

Reply:
Yeah I want to try and keep the story as simple as possible by having all the examples using the same immediately understandable pattern. 13.1 compatibility isn't something we should get hung up on, provided we can get this into 13.x by Jan.